jerry-core/ecma/builtin-objects/typedarray/ecma-builtin-typedarray-prototype.c

```diff
@@ -1601,6 +1601,190 @@ ecma_builtin_typedarray_prototype_find_index (ecma_value_t this_arg, /**< this a
   return ecma_builtin_typedarray_prototype_find_helper (this_arg, predicate, predicate_this_arg, false);
 } /* ecma_builtin_typedarray_prototype_find_index */
 
+/**
+ * The %TypedArray%.prototype object's 'copyWithin' routine helper
+ *
+ * @return ecma value
+ *         Returned value must be freed with ecma_free_value.
+ */
+static uint32_t
+ecma_builtin_typedarray_prototype_copy_within_helper(uint32_t length, /**< length of the typedarray */
```

Comment:
Could we make the method name a bit more informative on what does this helps to do? (and also add info about it in the comment above.